TestTools/StreamChatTestTools/SpyPattern/Spy/DatabaseContainer_Spy.swift

```diff
@@ -31,8 +31,7 @@ public final class DatabaseContainer_Spy: DatabaseContainer, Spy {
     private(set) var sessionMock: DatabaseSession_Mock?
 
     public convenience init(localCachingSettings: ChatClientConfig.LocalCaching? = nil) {
-        self.init(kind: .onDisk(databaseFileURL: .newTemporaryFileURL()), localCachingSettings: localCachingSettings)
```

Comment:
Really? 🫨 I thought we were using in-memory everywhere 😅

Reply:
This init is used a lot. Don't think there is real requirement for creating the actual DB file.

Reply:
Maybe for performance testing it is more realistic, but not a big deal I guess 🤔

Reply:
this is a good point actually, maybe we can keep it for the performance checks?

Reply:
I'll change it a bit so we use a file for performance tests

Reply:
I changed it to onDisk and these huge payload tests stopped working immediately. Don't understand still why it is so.